cmd/storj-sim/main.go

```diff
@@ -23,6 +23,11 @@ type Flags struct {
 	IsDev bool
 
 	OnlyEnv bool // only do things necessary for loading env vars
+
+	// Connection string for the satellite masterdb
+	SatelliteDB string
```

Comment:
A simpler design would allow to specify `--postgres postgres://storj:storj-pass@localhost/teststorj?sslmode=disable` and rest happens automatically.